test/integration_test/specs/exclude-resourcetype-migration/migration.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: stork.libopenstorage.org/v1alpha1
+kind: Migration
+metadata:
+  name: exclude-resourcetype
+spec:
+  # This should be the name of the cluster pair
+  clusterPair: remoteclusterpair
+  # If set to false this will migrate only the volumes. No PVCs, apps, etc will be migrated
+  includeResources: true
+  # If set to false, the deployments and stateful set replicas will be set to 0 on the destination. There will be an annotation with "stork.openstorage.org/migrationReplicas" to store the replica count from the source
+  startApplications: true
+  # 
+  excludeResourceTypes:
+  - Deployment
+  # List of namespaces to migrate
+  namespaces:
+  - mysql-1-pvc-exclude-resourcetype-migration
```

Comment:
nit: 
I think the namespace should be mysql-1-pvc-exclude-resourcetype-deployment

Reply:
done